packages/daemon/src/host.js

```diff
@@ -514,12 +523,39 @@ export const makeHostMaker = ({
       return cancelValue(formulaIdentifier, reason);
     };
 
+    /** @type {import('./types.js').EndoHost['gateway']} */
+    const gateway = async () => {
+      // TODO: we should only materialize local objects this way
+      return Far('Gateway', { provide: provideValueForFormulaIdentifier });
```

Comment:
This should probably be a singleton that the `gateway` method returns but does not construct.